packages/desktop-client/e2e/mobile.test.js

```diff
@@ -64,7 +64,7 @@ test.describe('Mobile', () => {
     const accountPage = await accountsPage.openNthAccount(0);
 
     await expect(accountPage.heading).toHaveText('Bank of America');
-    expect(await accountPage.getBalance()).toBeGreaterThan(0);
+    await expect(accountPage.balance).toHaveText('190.00');
```

Comment:
Waiting on `toBeGreaterThan` has early-returns. If the data is not yet loaded - in the UI we show `0` and the test fails.

Now we are matching against a specific text content. It will try to find this result and wait until the expectation times out. It makes this test case more stable.

Reply:
I think there is also a race condition here, we try to assert while the budget table is still loading. I attempted to fix that in this PR: https://github.com/actualbudget/actual/pull/3506.

Reply:
Oh that's great! I didn't notice you're also working on these tests. I'll pause my work until your PR is merged :)